includes/polling/mib/ceraos-mib.inc.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+$features = explode(' ', $device[features]);
+$num_radios = $features[0];
+
+$mib_oids = array();
+
+$radioNumber = 0;
+$IfIndex = 0;
+$ifIndexesArray = array();
+$ifIndexesArray = explode("\n", snmp_walk($device, "ifIndex", "-mIF-MIB -Oqv", ""));
```

Comment:
Can we not do something a bit better here where we pass the ifEntry cache from ports polling module to this module as we already have this info and I think polling modules are run in alpha numeric order.